erpnext/accounts/doctype/payment_request/payment_request.py

```diff
@@ -143,9 +144,10 @@ def create_payment_entry(self, submit=True):
 
 	def send_email(self):
 		"""send email with payment link"""
-		frappe.sendmail(recipients=self.email_to, sender=None, subject=self.subject,
-			message=self.get_message(), attachments=[frappe.attach_print(self.reference_doctype,
-			self.reference_name, file_name=self.reference_name, print_format=self.print_format)])
+		email_args = {"recipients":self.email_to, "sender":None, "subject":self.subject,
+			"message":self.get_message(), "attachments":[frappe.attach_print(self.reference_doctype,
+			self.reference_name, file_name=self.reference_name, print_format=self.print_format)]}
+		enqueue(method=frappe.sendmail, queue='short', timeout=300, async=True, now=True, **email_args)
```

Comment:
@manassolanki,

`now=True` will execute the sendmail method immediately, remove the now args.